cvat-ui/package.json

```diff
@@ -14,12 +18,15 @@
     "@babel/preset-env": "^7.6.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.6.0",
-    "@typescript-eslint/eslint-plugin": "^2.10.0",
+    "@typescript-eslint/eslint-plugin": "^2.19.2",
+    "@typescript-eslint/experimental-utils": "^2.19.2",
```

Comment:
I can't see where you use this package

Reply:
Yes, i think it was added, when i tried to run eslint local...
It looks like "babel": "^6.23.0" may be removed to, is not it?

Reply:
@nglazov

Yep, it looks like babel is also not needed.

Reply:
Fixed